recipes/xorg-xft/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set xorg_name = "libXft" %}
```

Comment:
D'oh. I think the ideal practice is to track the names of the X.org packages as closely as possible, in which case yes, this directory should have been named `xorg-libxft`.